Userland/Libraries/LibELF/DynamicLoader.cpp

```diff
@@ -227,7 +224,7 @@ void DynamicLoader::do_main_relocations()
             // GNU ld for some reason puts TLSDESC relocations into .rela.plt
             // https://sourceware.org/bugzilla/show_bug.cgi?id=28387
 
-            VERIFY(do_direct_relocation(relocation, cached_result, ShouldInitializeWeak::No, ShouldCallIfuncResolver::No) == RelocationResult::Success);
+            VERIFY(do_direct_relocation(relocation, cached_result, ShouldCallIfuncResolver::No) == RelocationResult::Success);
```

Comment:
pre-existing, but we shouldn't do things with side-effects in VERIFY() (273-274 does it 'correctly')